### ISSUE TITLE
Switching over and over to the same tooltype.

### ISSUE THREAD

**Issue by sl4shed** [alexfeed1990/AutoTools#1]
This is more of a technical issue. The system the mod uses to find out which tool to use (``itemstack.isToolEffective`` iirc) **only returns a boolean (true or false)**.

_For Example:_ If I want to mine a dirt block, if I have two shovels in my hotbar, the mod uses the ``isToolEffective`` function to determine what tools to use. In this case, both of my shovels return true. To my knowledge, since we modified the system we took the code from, there isn't really a way to fix this. The original code used ``getDestroySpeed(itemStack, blockState)`` which (i think) is a **legacy function** only left in the source code (of Minecraft) for mod support. 

The problem is, that this function, **_always returns 1.0F_**. This is problematic, since the mod **_will never find the correct slot to switch to_** if everything is effective (aka it will think that the current hotbar slot selected is the tool that should be used to mine the block). 

TL;DR: **We ~~stole~~ used some source code from somewhere and it used a legacy function which is broken. We switched it so that it works now but it has a technical limitation which causes this issue.**

_**I will look further into this issue and potential solutions when I can.**_
For now, you can leave solutions/suggestions you would think work in the comment section of this issue. Thank you for your time.
